Dokan/Dokan/Scenes/ProductDetails/Views/SellerInfo/Cell/SellerInfoCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SellerInfoCell.swift
+//  Dokan
+//
+//  Created by raniazeid on 17/08/2022.
+//
+
+import UIKit
+
+class SellerInfoCell: UICollectionViewCell {
+    // MARK: - outlets..
+
+    @IBOutlet private weak var sellerProductImage: UIImageView!
+    @IBOutlet private weak var productName: UILabel!
+    @IBOutlet private weak var productPrice: UILabel!
+    @IBOutlet private weak var productReviewRate: UILabel!
+    @IBOutlet private weak var productReviewerCount: UILabel!
+
+    // MARK: - life cycle...
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    // MARK: - func to configure cell with data
+
+    func configureCell(model: SellerInfoCellModel) {
+        sellerProductImage.setImage(with: model.sellerProductImage, placeholderImage: UIImage(named: "")!)
```

Comment:
```suggestion
        sellerProductImage.setImage(with: model.sellerProductImage, placeholderImage: UIImage(named: "")!)
```
Force unwrapping this will crash the app